main.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+
+function wait () {
+	echo "should we continue? (y|n)"
+	read -r user_input
+	if [ "$user_input" == 'y' ]
+	then
+		echo "continuing..."
+	else
+		echo -e "\e[1;42m Exiting... \e[0m"
+		exit 0
+	fi
+}
+function should () {
+	echo "should I \"$1\" (y|n|e)"
+	read -r user_input
+	if [ "$user_input" == 'y' ]
+	then
+		$1
+		status
+	elif [ "$user_input" == 'n' ]
+	then
+		echo -e "\e[1;43m Skipping... \e[0m"
```

Comment:
some of the text is not very readable e.g. white font on yellow or light green background 
perhaps flipping the font to black would help?